src/patternfly/layouts/Grid/styles.scss

```diff
@@ -0,0 +1,93 @@
+// URL.com/guidelines#layout
+// A layout doesn't have variables
+@import "../../patternfly-utilities";
+
+.pf-l-grid {
+  display: flex;
+  flex: 1;
+  flex-flow: row wrap;
+  padding: 0;
+  margin: 0;
+  @supports (display: grid) {
+    display: grid;
+    grid-template-columns: repeat(12, [col-start] 1fr);
+  }
+
+  // Nested grids
+  .pf-l-grid {
+    display: flex;
+    @supports (display: grid) {
+      display: grid;
+    }
+  }
+
+  &__item {
+    @supports (display: grid) {
+      margin: 0;
+    }
+  }
+
+  > * {
+    flex-basis: 100%;
+    @supports (display: grid) {
+      flex-basis: 0;
+      grid-column-start: auto;
+      grid-column-end: span 12;
+    }
+  }
+
+  // Generate default modifier classes
+  $breakpoints: (
+    "": null,
+    "xs": "-on-xs",
+    "sm": "-on-sm",
+    "md": "-on-md",
+    "lg": "-on-lg",
+    "xl": "-on-xl"
+  );
+
+  // Loop through $breakpoints map to generate responsive classes
+  @each $breakpoint, $value in $breakpoints {
+
+    @include GridMediaQuery($breakpoint) {
+      @include FlexboxFallback($value);
+      @include FlexboxWithGuttersFallback($value);
+      @include GridItemModifiers($value);
+    }
+  }
+
+  &.pf-has-gutters {
+    margin: -#{$pf-global--gutter/2};
+    > * {
+      margin: #{$pf-global--gutter/2};
+    }
+  }
+
+  @supports (display: grid) {
+    &.pf-has-gutters {
+      margin: unset;
+      grid-gap: $pf-global--gutter;
```

Comment:
same here with grid gap. Local variables are defined and always reference a global variable.

Reply:
This should be:
```scss
$pf-global--has-gutter--GridGap: $pf-global--gutter !default;
```